lib/WorseReflection/Bridge/TolerantParser/Reflection/AbstractReflectionClass.php

```diff
@@ -43,9 +37,6 @@ public function isEnum(): bool
         return $this instanceof ReflectionEnum;
     }
 
-    /**
-     * @deprecated Use instanceof instead
-     */
     public function isClass(): bool
```

Comment:
why did we remove these deprecations?

Reply:
The deprecations have been moved to the interface, because if you implement the interface you should be aware of that. (I know we only have one implementation right now.) But from my phpstorm days, having documentation on the interfaces is much more helpful since everywhere else in the project you are just using the interface (which doesn't have the deprecations visible).